### PR TITLE
Add 1.1.1 issue template and input manifests

### PR DESCRIPTION
**Pull request by peterzhuamazon**
### Description
Add 1.1.1 issue template and input manifests.
This is an improvement of the now closed #1100.
 
### Issues Resolved
Related to #870
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:7]: Can we rename the these files to `components`?  Its the language used throughout the release process.  Plugin is a kind of component, but not all components are plugins like common-util

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:32]:
```suggestion
- [ ] If including changes in this release, increment the version on 1.1 branch to `1.1.1`, otherwise stay on `1.1.0`.
```

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:22]: Can we remove this line - if we have nodes we should integrate them into the how to use section?  If we need teams to take different action then we should call it out in the task list below

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:32]: Rather than add annotations for if/else in inline, what do you think about a section "What should I do if my plugin isn't making any changes?" that lays out exactly what to checkout off, including this line item.  This was part of our retro feedback on 1.2.0

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:38]: `All code changes for 1.1.1 are complete.` isn't this line the same as that one?

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:39]: Encouraging direct contact to the release owner isn't going to scale, making sure this process can be followed without any direct contact.  A link to an example would go along way here to make sure folks are following through consistently would be great.

```suggestion
- [ ] Complete integration and sanity tests, and update results in the comment, [example](link).
```

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:40]: Great callout, this exposes an issue, how do they wait and test again?  every day, every hour?  Or should they get a notice about a release candidate build somehow?  We used this issue, https://github.com/opensearch-project/opensearch-build/issues/954 for when we had to 'change the plan' on everyone, maybe we could cut a ticket for 'testing has completed against the RC build? in a similar way'

Here is what I would recommend, lets keep 'Find/fix bugs using latest docker image', and lets add a new line for 'completed release candidate testing build #TBD`, then while slightly painful, we could add a new line for each additional RC build that arrives with the link on where to get it.

File: .github/meta_issue_template/release-1.3.0-plugin-issues.md
Comment (peternied) [inline .github/meta_issue_template/release-1.3.0-plugin-issues.md:1]: Did you mean to include this?  We are still waiting on the retro feedback for 1.2.0

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:1]: What is the usage of this `-min` vs the other issue?

Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:1]: It is only used for OpenSearch and Dashboards core repo.
Since they do not need any changes they also do not need any specific tests.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:7]: Will do.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:22]: Sure.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:1]: Can we adapt the single template to cover this case in the with a "What to do if you have no changes section".  This second template requires more work by the release manager that it would be better to avoid.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:32]: Fixed.

Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:32]: Good idea.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:38]: This is a second confirmation just to make sure component owners have all the changes ready.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:39]: I totally get that but at the moment after viewing through several issues I cannot find a good example, as component owners would just check CI results or do manual testing.

Would you think adding my side of the integTest post would help?
https://github.com/opensearch-project/opensearch-build/issues/1118

Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:39]: Anyway I will change it with a link to my issue, and extend from there later on.
Thanks.

File: .github/meta_issue_template/release-1.3.0-plugin-issues.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.3.0-plugin-issues.md:1]: We can leave it here and update later.

Comment (peternied) [reply .github/meta_issue_template/release-1.3.0-plugin-issues.md:1]: It is basically a duplicate of `release-1.1.1-plugin-issues.md` I don't see a point in reviewing it, if we are going to copy/paste on-top of it, might as well do it later.

File: src/ci_workflow/ci_input_manifest.py
Comment (dblock) [inline src/ci_workflow/ci_input_manifest.py:22]: should be `build.filename`

File: src/ci_workflow/ci_check_manifest_component.py
Comment (dblock) [inline src/ci_workflow/ci_check_manifest_component.py:22]: Can it be `self.target.platform` instead of hard-coded `linux`?

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:39]: Adding a link to a `RELEASE_PROCESS.MD#HowToPublishTestResults` that we can fill in later works too.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:38]: Rather than `merge` what do you think about making it 'confirmation' oriented?
```suggestion
- [ ] Confirm that all changes for 1.1.1 have been merged.
```

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md:40]: Added, thanks.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md]: Where is this file exist on the repo?
RELEASE_PROCESS.MD#HowToPublishTestResults

File: src/ci_workflow/ci_input_manifest.py
Comment (peterzhuamazon) [reply src/ci_workflow/ci_input_manifest.py]: fixed.

File: src/ci_workflow/ci_check_manifest_component.py
Comment (peterzhuamazon) [reply src/ci_workflow/ci_check_manifest_component.py]: This `target` is `CiTarget`, which does not have the `platform` var.

**Review by dblock — APPROVED** [review]

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md]: It doesn't exist yet, but it's a great fast follow

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peternied) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md]: [Blocking] I think this instruction is confusing for teams, please update.

File: .github/meta_issue_template/release-1.1.1-component-issues-min.md
Comment (peternied) [inline .github/meta_issue_template/release-1.1.1-component-issues-min.md:29]: Note; I mean to pull this into the `How to use this issue section` not to create a new section in the checklist with instructions of how to use the checklist

**Review by peternied — CHANGES_REQUESTED** [review]
I assume we are going to cut the 1.1.1 components issues as soon as this is merged - so I request that the first `pre-release` instruction is changed per that comment thread, all other feedback is at your discretion.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md]: Fixed.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md]: Got it.

File: .github/meta_issue_template/release-1.1.1-plugin-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md]: Fixed.

Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-plugin-issues-min.md]:
```
- [ ] If including changes in this release, increment the version on 1.1 branch to `1.1.1` for Min/Core, and `1.1.1.0` for components. Otherwise, keep the version number unchanged for both.
```

**Review by peternied — APPROVED** [review]

File: .github/meta_issue_template/release-1.1.1-component-issues-min.md
Comment (peterzhuamazon) [reply .github/meta_issue_template/release-1.1.1-component-issues-min.md]: Added.

**Pull request merged**
